src/app/_components/KeyMemberCard.tsx

```diff
@@ -0,0 +1,48 @@
+import Image from 'next/image'
+
+import { LinkedinLogo } from '@phosphor-icons/react/dist/ssr'
+
+import { Heading } from '@/components/Heading'
+import { Icon } from '@/components/Icon'
+import { TextLink } from '@/components/TextLink'
+
+import type { MemberData } from '@/types/memberType'
+
+type KeyMemberCardProps = MemberData
+
+export function KeyMemberCard({
+  name,
+  title,
+  linkedin,
+  image,
+}: KeyMemberCardProps) {
+  return (
+    <li className="flex rounded-lg border border-brand-500 p-1">
+      <Image
+        src={image}
+        alt={`Photo of ${name}`}
+        sizes="10rem"
+        className="aspect-[3/4] w-32 rounded object-cover"
+      />
+
+      <div className="relative m-3 grow">
+        <Heading isClamped tag="h3" variant="lg">
```

Comment:
We shouldn't clamp the heading here - Since there's no link to, let's say, another page where the user can see it in full - Like for blog posts or ecosystem projects or events that have their own individual pages.

Reply:
✅